### DISCUSSION TITLE
How can I save a predictor for using it again

### DISCUSSION THREAD

**Discussion by MarleneBs** [awslabs/gluonts#2418] [Q&A]
On the page it is said with Path how to save a predictor, but then also comes the warning that you can no longer use it somehow. Can you save the model so that you can open it in another notebook and use it with other data?

**Comment by jaheba** [comment]
Hi, can you share the warning?

Generally, you can use `predictor.serialize(path)` to store the predictor, and `Predictor.deserialize(path)` to load it again.

Here is an example:

https://github.com/awslabs/gluonts/blob/4f423add2709eabf12f2481bb49fd9c53552676f/test/mx/model/simple_feedforward/test_serde.py#L50-L52

**Reply by MarleneBs** [reply]
WARNING: root: Serializing RepresentableBlockPredictor instances does not save the prediction network stucture in a backwards-compatible manner. Be careful not to use this method in production
